grunt-tasks/jst.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function(grunt) {
+  grunt.config("jst", {
+    compile: {
+      options: {
+        namespace: "wcTmpl",
+        prettify: true,
+        processContent: function(src) {
+          // strip the opening and closing <script> tags...
+          // otherwise, the template functions will just inject script elements
+          // that won't render.
+          src = src.split(/\<script type=['"]text\/template['"]>/)[1];
```

Comment:
string.split as a poor man's parser.... seemed nicer than actually using regexp to parse this. Alternative would be to pull in a [js-dom](https://www.npmjs.com/package/jsdom)-like package and parse the script contents and replace it with `ParentNode.firstElementChild()` or something.